src/gnssro/gnss_ztd_ground_based.py

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env python3
+
+#
+# (C) Copyright 2019-2024 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+#
+# decode TENET formatted ground-based GNSS Total Electron Content (TEC)
+#
+
+import sys
+import os
+import time
+from datetime import datetime, timezone
+import numpy as np
+import logging
+
+# These modules need the path to lib-python modules
+import pyiodaconv.ioda_conv_engines as iconv
+from pyiodaconv.orddicts import DefaultOrderedDict
+from pyiodaconv.def_jedi_utils import iso8601_string, epoch
+
+os.environ["TZ"] = "UTC"
+
+# these are the unique values in the raw input file
+varDict = {
+    'zenithTotalDelay': ['zenithTotalDelay', "float", 'm'],
+}
+# extend the variable keys including 'Error', and 'Flag'
+extended_varDict_keys = set()
+for key in varDict.keys():
+    extended_varDict_keys.update([key, key + 'Error', key + 'Flag'])
+
+# these are the MetaData common to each input
+locationKeyList = [
+    ('latitude', 'float', 'degrees_north'),
+    ('longitude', 'float', 'degrees_east'),
+    ('dateTime', 'long', iso8601_string),
+#   ('stationIdentifierWMO', 'integer', 'WMO assigned number for the site'),
+    ('stationIdentifier', 'string', 'GNSS ground-based receiving station name'),
```

Comment:
```suggestion
    ('stationIdentifier', 'string', 'GNSS ground-based receiving station name'),
    ('stationElevation', 'string', 'GNSS ground-based receiving station name'),
```

we'll need stationElevation too!